nginx/openflights

```diff
@@ -24,27 +24,21 @@ server {
 	rewrite "^/banner/(.+).png$" /badge/banner.php?user=$1;
     }
 
-    if ($uri !~ \.) {
```

Comment:
I'm not quite sure what this is supposed to be preventing...

It does a negated regex match looking for a literal `.`... But when usernames can have `.` in them..

Maybe airline and airport shouldn't be in it, but what does that give?

Reply:
@jpatokal Can you remember what you added this for?

Reply:
My best guess would be that it's trying to avoid rewriting file paths that already have .php/html, but the regexes should be specific enough to avoid that.